scripts/post_install/stub_generation.py

```diff
@@ -33,102 +32,166 @@ def __AddSubmodules(output_path: Path):
             __AddSubmodules(cpp_sub_module_path)
 
 
-def __binary_module_from_name(module_name: str) -> str:
-    """Get the binary module's stub from its name."""
-    return "Kratos" + module_name
+def __FindCPPModuleImportsInPythonModules(current_path: Path, cpp_module_names_list: list[str], cpp_python_modules_dict: dict[str, Path]):

```

Comment:
type hints with arguments (eg.: ```list[str]```) are a python 3.9 feature. I usually put them in quotes until we drop support for older python versions (```"list[str]"```).
```suggestion
def __FindCPPModuleImportsInPythonModules(current_path: Path, cpp_module_names_list: "list[str]", cpp_python_modules_dict: "dict[str, Path]") -> None:

```